src/Tizen.NUI/src/public/Accessibility/Accessibility.cs

```diff
@@ -332,7 +332,8 @@ public enum SayFinishedState
 
         private static void SayFinishedEventCallback(string status)
         {
-            var result = sayFinishedStateDictionary.GetValueOrDefault(status, SayFinishedState.Invalid);
+            SayFinishedState result = SayFinishedState.Invalid;
+            sayFinishedStateDictionary.TryGetValue(status, out result);
```

Comment:
I think that `result` would be set by `TryGetValue` in all cases, so perhaps the assignment of `SayFinishedState.Invalid` should be done after `TryGetValue`.

The [docs](https://learn.microsoft.com/en-us/dotnet/api/system.collections.generic.dictionary-2.trygetvalue) say:

> When this method returns, contains the value associated with the specified key, if the key is found; otherwise, the default value for the type of the value parameter. This parameter is passed uninitialized.

Reply:
Wow! I didn't know detail of that! I fix it now. Thank you for your comments.